packages/editor/src/classes/MeshCombinationGroup.ts

```diff
@@ -74,7 +78,7 @@ async function meshStandardMaterialComparator(group, a, b) {
     a.aoMapIntensity === b.aoMapIntensity &&
     a.normalScale.equals(b.normalScale) &&
     a.emissive.equals(b.emissive) &&
-    (await meshBasicMaterialComparator(group, a, b)) &&
+    (await meshBasicMaterialComparator(group, a as any, b as any)) &&
```

Comment:
If these promises aren't codependent we can use `await Promise.all([promise1,promise2])` that would be a lot quicker